SimpleColorApp/SimpleColorApp/ContentView.swift

```diff
@@ -8,17 +8,56 @@
 import SwiftUI
 
 struct ContentView: View {
-    var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
-            Text("Hello, world!")
-        }
-        .padding()
+  
+  @State private var redSliderValue: Double
+  @State private var greenSliderValue: Double
+  @State private var blueSliderValue: Double
+  @State private var boxColor:Color
+  
+  init() {
+    let defaultRed = 200.0
+    let defaultGreen = 100.0
+    let defaultBlue = 240.0
+    
+    self.redSliderValue = defaultRed
+    self.greenSliderValue = defaultGreen
+    self.blueSliderValue = defaultBlue
+    self.boxColor = ContentView.calculateBoxColor(red: defaultRed, green: defaultGreen, blue: defaultBlue)
+  }
+  
+  var body: some View {
+    VStack {
+      Text("Color Picker")
+        .font(.largeTitle)
+      RoundedRectangle(cornerRadius: 0)
+        .fill(boxColor)
+      Text("Red")
+      HStack {
+        Slider(value: $redSliderValue, in: 1.0...255.0)
```

Comment:
The instructions kind of lead you down this path of creating a range of 1...255 but by default `Slider` ranges from 0.0...1.0 which matches how the color values work. So you can save yourself some conversion logic if you just use the defaults and then just convert to 255 and `Int` for display. Something to tinker with if you're interested.